python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/vlan-interfaces.j2

```diff
@@ -315,6 +315,65 @@ interface {{ vlan_interface.name }}
 {%     if vlan_interface.isis_network_point_to_point is arista.avd.defined(true) %}
    isis network point-to-point
 {%     endif %}
+{%     if vlan_interface.isis_authentication_mode is arista.avd.defined %}
+{%         set auth_cli = "isis authentication mode" ~ " " ~ vlan_interface.isis_authentication_mode %}
+{%         if vlan_interface.isis_authentication_mode == "sha" and vlan_interface.isis_authentication_mode_sha is arista.avd.defined %}
+{%             if vlan_interface.isis_authentication_mode_sha.both.key_id is arista.avd.defined %}
+{%                 set auth_cli = auth_cli ~ " key-id " ~ vlan_interface.isis_authentication_mode_sha.both.key_id %}
+{%                 if vlan_interface.isis_authentication_mode_sha.both.rx_disabled is arista.avd.defined %}
+{%                     set auth_cli = auth_cli ~ " rx-disabled" %}
+{%                 endif %}
+   {{ auth_cli }}
+{%             else %}
+{%                 if vlan_interface.isis_authentication_mode_sha.level_1.key_id is arista.avd.defined %}
```

Comment:
```suggestion
{%             elif vlan_interface.isis_authentication_mode_sha.level_1.key_id is arista.avd.defined %}
```
same for level_2 , correct the indentation after doing this.

Reply:
If both is not set then only level1 and level2 are configured. So added else condition after both.

Reply:
Okay it will get converted as `both` config in CLI